app/controllers/api/v1/analytics/mrrs_controller.rb

```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+module Api
+  module V1
+    module Analytics
+      class MrrsController < Api::BaseController
+        before_action :authorize
+
+        def index
+          result = ::Analytics::Mrr.find_all_by(current_organization.id, **filters)
+
+          render(
+            json: ::CollectionSerializer.new(
+              result,
+              ::V1::Analytics::MrrSerializer,
+              collection_name: 'mrrs',
+            ),
+          )
+        end
+
+        private
+
+        def filters
+          {
+            currency: params[:currency]&.upcase,
+            months: params[:months].to_i,
+          }
+        end
+
+        def authorize
+          forbidden_error(code: 'premium_feature') unless License.premium?
```

Comment:
I think that it would be good to follow common pattern here. Usually license check is placed in service, so that we can simply return `result.forbidden_failure!`. In that case error code will be set to `feature_unavailable` by default. In controller you can simply check if service result is success and return r`ender_error_response(result)` if it is not successful

Reply:
@lovrocolic 
cc: @ansmonjol 
Lets move it to a service but I'd recommend to do it in a separate refactoring PR after this one's merged. 
This change would require 2 other (already merged) PRs to follow this as well.

Reply:
@lovrocolic I added the service in this PR but I'll another PR to fix the rest.